tests/testthat/test-contribution_matrix.R

```diff
@@ -0,0 +1,244 @@
+
+test_that("CreateVMatrix() works for binary outcomes", {
+  data <- data.frame(Study = c("A", "A", "B", "B", "C", "C", "C", "D", "D"),
+                     T = c(1, 2, 2, 3, 1, 2, 3, 3, 4),
+                     Treatment = c("Hydrogen", "Oxygen", "Oxygen", "Sulphur", "Hydrogen", "Oxygen", "Sulphur", "Sulphur", "Zinc"),
+                     R = c(8, 13, 14, 21, 10, 15, 22, 23, 30),
+                     N = c(40, 41, 42, 43, 44, 45, 46, 47, 48))
+  
+  studies <- c("A", "B", "C", "D")
+  treatments <- c("Hydrogen", "Oxygen", "Sulphur", "Zinc")
+  outcome_type <- "Binary"
+
+  #Variance formula from "Introducion to Meta-Analysis" 2nd edition, Borenstein et al, page 36.
+  effect_variances <- vector(length=5)
+  effect_variances[1] <- 1/8 + 1/(40-8) + 1/13 + 1/(41-13)
+  effect_variances[2] <- 1/14 + 1/(42-14) + 1/21 + 1/(43-21)
+  effect_variances[3] <- 1/10 + 1/(44-10) + 1/15 + 1/(45-15)
+  effect_variances[4] <- 1/10 + 1/(44-10) + 1/22 + 1/(46-22)
+  effect_variances[5] <- 1/23 + 1/(47-23) + 1/30 + 1/(48-30)
+  V_matrix <- diag(effect_variances)
```

Comment:
Can you change the names in all of these test cases to be something like `expected_v_matrix` please? Again, it just helps with readability.